backend/src/case_file/case_file.service.ts

```diff
@@ -2072,6 +2073,91 @@ export class CaseFileService {
       }
     };
 
+    //--
+    //-- determines what type of action to apply to the actions
+    //--
+    const _derp = async (
```

Comment:
this method should be given a proper name that better represents what it does.

Reply:
gah, totally missed that. Made the requested updates to for this, as well as cleaning up some commented out code.